experiment/tracer/static/script.js

```diff
@@ -0,0 +1,273 @@
+'use strict';
+
+var logger = {
```

Comment:
I would assume the tracer is a new feature to the dashboard. Do you think should we start migrating code to ES6?

Reply:
Eventually, I would love to see this as part of deck. Since the tracer is still a new thing, (and I still need to figure out how to get an oauth token in JS) let's keep this work here for now.

Reply:
@kargakis ?
@qhuynh96 isn't talking about moving the dashboard, just using shiny new ECMAScript 6 features [not sure what or why though :-)] (?)

Agree that this should be part of Deck though, I'd like to keep all the public facing web APIs / content in Deck if we can.

Reply:
> I'd like to keep all the public facing web APIs / content in Deck if we can.

The only issue with this is that it's a web API meant to be used by prow admins. Net-new stuff compared to the rest of the deck pages. That's why we need to sort out securing it first, and I need to find cycles and explore this.

We are running the tracer backend today behind an oauth proxy[1][2][3][4] and the only thing remaining is to figure out how to oauth from this dashboard.

[1] https://tracer-ci.svc.ci.openshift.org/
[2] https://github.com/openshift/oauth-proxy#openshift-oauth-proxy
[3] https://github.com/openshift/release/blob/f0f027251a419ebaaf73a0c06af5e2dd861ee1d9/cluster/ci/config/prow/openshift/tracer.yaml#L53-L78
[4] https://github.com/openshift/release/blob/f0f027251a419ebaaf73a0c06af5e2dd861ee1d9/cluster/ci/config/prow/openshift/tracer_rbac.yaml#L12

Reply:
Right, that doesn't sound "public facing" just yet :-)